script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -141,62 +155,71 @@ The sound of rain can be heard outside.
 [sha:a74568f95af872e0a795381dafe21e2464e0b72e]{
 -- Page 253, Offset 5497.
 -- 「……先輩、起きてるんですか……？」
-"...... Senpai, are you awake ......?"
+"Senpai... You were awake...?"
 }
 [sha:772637e12b76647094fdf2858d92e4953f506e00]{
 -- Page 254, Offset 5498.
 -- 「はい。遠野くんが眠るまで眠れません。わたし、いちおう女の子なんですから」
-"Yes. I can't sleep until you're asleep. I'm a girl after all."
+"Yes. I can't sleep until you're asleep.
+ Despite everything I %{i}am%{/i} still a girl."
 }
 [sha:1fabba2dbd0a819e0b2e9ed81c73edefc7e0c03b]{
 -- Page 254, Offset 5499.
 -- 「……ごめん。やっぱり台所で眠るよ、俺」
-"...... Sorry. I'll sleep in the kitchen after all."
+"Sorry... Maybe I'd better go sleep in the kitchen after all."
 }
 [sha:e0ac4e53f0286fe455bfd4695e604b9e468e24f5]{
 -- Page 255, Offset 5500.
 -- 「もうっ、何度も言わせないでください。遠野くんは風邪の引き始めなんですから、台所には寝かせられません。いいからイヤなことは忘れて眠っちゃってください」
-"Damn it, don't make me say it again. You are in the early stages of a cold and I can't let you sleep in the kitchen. Just forget about it and go to sleep."
+"Damn it, don't make me say it again.
+ You're in the early stages of catching a cold, I'm not about to let you sleep in the kitchen.
+ Just forget about whatever it is that's bothering you and go to sleep."
 }
 [sha:b56ed44056da699f757fc980b30bd062442730b3]{
 -- Page 256, Offset 5501.
 -- 「―――――厭な、こと」
-"―――I... can't."
+// Tempted to reify forget here
+"What's... bothering me..."
 }
 [sha:fde17e7b8e444dd4044d2b27b47a7dc93b3efbe8]{
 -- Page 256, Offset 5502.
 -- ……ああ、それは無理だ。
-......Ahh, it's impossible.
+...Ahh, that's impossible.
 }
 [sha:e32c4e9e038b7b1476da69943610cb945a26a891]{
 -- Page 256, Offset 5503.
 -- あれは忘れられないものだし、忘れてはいけない事だ。
-It's not something I can just forget, it's not something I %{i}should%{/i} forget about.
+It isn't something I can forget.
```

Comment:
This part bugs me a bit. I feel like an 'in fact,' should be added to the start of the second line.